ilastik/workflows/tracking/conservation/conservationTrackingWorkflow.py

```diff
@@ -391,9 +391,9 @@ def post_process_lane_export(self, lane_index, time=1):
                     logger.error("Cannot export from plugin with empty output filename")
                     return
 
-                if time == 0 and selectedPlugin == 'H5-Event-Sequence' and os.path.exists(filename + '/H5-Event-Sequence/'):
+                if overwriteExistingFiles == False and selectedPlugin == 'H5-Event-Sequence' and os.path.exists(filename + '/H5-Event-Sequence/'):
```

Comment:
You could even write `if not overwriteExistingFiles and ...` now 😉 

Now that I think about it, the workflow file is not the right place for these checks. The bad thing is that we need an if statement for every new plugin here. The plugin itself knows best where it tries to save the results, so we could push this `exportStatus` one level further: 
Plugins should get an extra method to check whether they would overwrite any files, and that could be called from within `self.trackingApplet.topLevelOperator.getLane(lane_index).exportPlugin(filename, exportPlugin` **`, overwriteExistingFiles)`**. What do you think about that @LetiP?